figures/S1.supplemental_table1/notebooks/Stable1.ipynb

```diff
@@ -0,0 +1,1093 @@
+{
+ "cells": [
```

Comment:
Consider adding a header cell with a title and short description of what the notebook is for.

Reply:
added!